scripts/EGAP/files_to_import_structure.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+import argparse
+from distutils.dir_util import copy_tree
+
+get_item_id = lambda _path: _path.split(os.sep)[-1].split(' ')[0]
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-source', '--source', help='')
+parser.add_argument('-destination', '--destination', help='')
+
+
+def main(origin_path, new_path):
+    # Copy whole tree, then pick out anonymous
+    for item in os.listdir(origin_path):
+        item_id = get_item_id(item)
+        source = os.path.join(origin_path, item)
+        destination = os.path.join(origin_path, item_id, 'data', 'nonanonymous')
+        copy_tree(source, destination)
+
+    for root, dir, files in os.walk(new_path):
+        for item in files:
+            source = os.path.join(root, item)
+            if 'PAP_anon' in item or 'Anonymous.' in item:
+                destination_parent = os.path.join('/'.join(root.split('/')[:-1]), 'anonymous')
```

Comment:
Could you break your non-directory-walking logic into functions and unit test those? If we end up having to do complicated logic for determining which files to keep and which not to, or if the anonymous stuff becomes trickier, I'd like to have those items well tested.